### DISCUSSION TITLE
Changing the name of an input file will affect the results ?!

### DISCUSSION THREAD

**Discussion by Xelver** [idaholab/moose#16216] [Q&A General] [answered]
Hey guys,

I've met a very weird problem recently.

I built an input file named '**test**.i' and it could run well. 
However, when I changed its name to '**test_th.i**', there was an error 'segmental fault'.
If I change its name to '**test_t.i**', it ran well again.... (The input file is attached) 
[test_t.txt](https://github.com/idaholab/moose/files/5550674/test_t.txt)

I find the problem very tricky...

The version of my MOOSE is: (Ubuntu 16.04)
commit db9be3fcd570a2a063a39cadc448fc9b83c696a7

Merge: 07f96ea 6984549

Author: moosetest <bou...@inl.gov>

Date:   Mon Mar 11 12:38:45 2019 -0600   

Thanks a lot 

Fengrui Xiang

**Comment by dschwen** [comment, marked as answer]
It might help to see a list of all files in the directory you're running the simulation in.

**Reply by Xelver** [reply]
Thanks for your reply.
Actually, there's nothing special in the directory. I just created a new App and activated the HeatCondution Modulus.

**Comment by GiudGiud** [comment]
Closing as stale.
For the record, changing the name of the input file does not influence results.
